app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb

```diff
@@ -193,6 +194,20 @@ def self.filter_for_coc_codes(coc_codes)
       current_scope.where(project_id: project_ids)
     end
 
+    def self.filter_for_dv_status(filter)
+      return current_scope if filter.dv_status.blank?
+
+      client_ids = GrdaWarehouse::ServiceHistoryEnrollment.entry.joins(:project, enrollment: :health_and_dvs).
+        merge(GrdaWarehouse::Hud::Project.where(id: filter.anded_effective_project_ids)).
+        merge(
+          GrdaWarehouse::Hud::HealthAndDv.where(
+            InformationDate: filter.range,
+            DomesticViolenceVictim: filter.dv_status,
```

Comment:
Flag this so we remember to change it for FY2024?

Reply:
Thanks!